.ci/magic-modules/create-pr.sh

```diff
@@ -33,15 +36,59 @@ $(git log -1 --pretty=%B)
 EOF
 
   git checkout -b "$BRANCH_NAME"
-  TF_PR=$(hub pull-request -b "$TERRAFORM_REPO:master" -F ./downstream_body)
+  if TF_PR=$(hub pull-request -b "$TERRAFORM_REPO:master" -F ./downstream_body); then
+    DEPENDENCIES="${DEPENDENCIES}depends: $TF_PR ${NEWLINE}"
+  else
+    echo "Terraform - did not generate a PR."
+  fi
   popd
 
-  cat << EOF > ./pr_comment 
+  pushd build/puppet/sql
+  cat << EOF > ./downstream_body
+$(git log -1 --pretty=%B)
+
+<!-- This change is generated by MagicModules. -->
+EOF
+
+  git checkout -b "$BRANCH_NAME"
+  if PUP_SQL_PR=$(hub pull-request -b "$PUPPET_SQL_REPO:master" -F ./downstream_body); then
+    DEPENDENCIES="${DEPENDENCIES}depends: $PUP_SQL_PR ${NEWLINE}"
+  else
+    echo "Puppet SQL - did not generate a PR."
+  fi
+  popd
+
+  pushd build/puppet/compute
+  cat << EOF > ./downstream_body
+$(git log -1 --pretty=%B)
+
+<!-- This change is generated by MagicModules. -->
+EOF
+
+  git checkout -b "$BRANCH_NAME"
+  if PUP_COMPUTE_PR=$(hub pull-request -b "$PUPPET_COMPUTE_REPO:master" -F ./downstream_body); then
+    DEPENDENCIES="${DEPENDENCIES}depends: $PUP_COMPUTE_PR ${NEWLINE}"
+  else
+    echo "Puppet Compute - did not generate a PR."
+  fi
+  popd
+
+  if [ -z "$DEPENDENCIES" ]; then
+    cat << EOF > ./pr_comment 
+I am a robot that works on MagicModules PRs!
+I checked the downstream repositories, and none of them seem to have any changes.
```

Comment:
Neat feature.

The message however implies that we checked all the downstream repositories. Consider changing the wording and including a link to `.ci/README.md` and update `.ci/README.md` to include the list of downstream repos the magician is checking.

As you add support for more downstream repos, it allows us to easily track what was supported when the PR was created.

Reply:
Done!